ports/superhaxagon/port.json

```
@@ -0,0 +1,40 @@
+{
+  "version": 4,
+  "name": "superhaxagon.zip",
+  "items": [
+    "SuperHaxagon.sh",
+    "superhaxagon/"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "SuperHaxagon",
+    "porter": [
+      "RedTopper"
+    ],
+    "image": {
+      "screenshot": "screenshot.png",
+      "covers": [
+        "cover.png"
+      ],
+      "thumbnail": null,
+      "video": null
+    },
+    "desc": "SuperHaxagon is a cross platform, open source Super Hexagon clone. Avoid the falling walls and survive for as long as possible in this fast-paced arcade game!",
+    "desc_md": null,
+    "inst": "This game is Ready to Run.",
+    "inst_md": null,
+    "genres": [
+      "arcade"
+    ],
+    "rtr": true,
+    "exp": false,
+    "runtime": [],
+    "store": [],
+    "free": true,
```

We are changing to `"availability"` over "free".
```suggestion
    "availability": "full",
```

---

Thanks! I'll make sure to update this in my own personal repo too.